src/diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -53,7 +53,12 @@ def custom_error(self, msg):
                 "Use --help for help.",
             ]
         ),
-        epilog="Please report bugs to diffpy-dev@googlegroups.com.",
+        epilog="\n".join(
+            [
+                "For more information, see the PDFmorph manual."
+                "Please report bugs to diffpy-dev@googlegroups.com.",
```

Comment:
we don't use this any more so it would be good to remove diffpy-dev from everywhere.  Let's ask them to report questions and bugs to diffpy-users.

Can we also do a global search and make this change everywhere in all the projects.

Reply:
Changed everywhere in `PDFmorph`. I put in an [issue](https://github.com/bg-mission-control/operation-release-everything/issues/6) to `operation-release-everything` about the remaining instances of `diffpy-dev` that need to be updated.